### DISCUSSION TITLE
如何使用externalGlobals配置打包tiptap的引用包？

### DISCUSSION THREAD

**Discussion by WhiteSevs** [lisonge/vite-plugin-monkey#165] [Q&A] [answered]
[https://tiptap.dev/docs/editor/installation/vue3](https://tiptap.dev/docs/editor/installation/vue3)

看了一下`@tiptap/pm`内是这样的
![Clip_2024-06-07_19-39-18](https://github.com/lisonge/vite-plugin-monkey/assets/50544447/7f7697ec-edc4-4afd-8457-2a60876d575b)

**Comment by lisonge** [comment, marked as answer]
tiptap 没有提供 iife/umd 格式的文件，无法配置

**Comment by WhiteSevs** [comment]
想了下要缩小打包后的体积只能使用<script type="module">加载到页面中了
